.env

```diff
@@ -24,3 +24,4 @@ DATA_VOLUME_CONTAINER=/data
 
 # Callback url for Auth0. Override if using non-production server
 AUTH0_CALLBACK_URL=https://jupyter.4dnucleome.org/hub/oauth_callback
+ADMIN_EMAILS=william_ronchetti@hms.harvard.edu
```

Comment:
Can we start a confluence page where we remember where we have these things? I guess we could search all our repos for your name if you ever leave, but it feels like it should be more orderly than that.